randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -1713,18 +1713,28 @@ Extra - asset_id: collision_camera_019
       Screw Attack
   > Event - Grapple Block (grapplepulloff1x2_000)
       All of the following:
-          Morph Ball
+          Grapple Beam and Morph Ball
           Any of the following:
               Cross Bomb
               All of the following:
                   # Bomb & Power Bombs, to desync the tunnel bomb blocks from the ledge bomb blocks
                   Bomb and Power Bombs
+              All of the following:
+                  # Power bombs (video: https://twitter.com/i/status/1545111396760690692 )
+                  Power Bombs ≥ 2
```

Comment:
This should use some sort of movement trick for taking up the spot of the bomb block so it doesnt break.

Reply:
I agree. Either movement or knowledge should be added. How would you rate the skill level needed for this movement? I do not want to rate skill level since I just got here.

Reply:
Knowledge is not needed. It's vanilla game knowledge to know this area is here. 1 and 2 can be trickless. 3 I would personally mark as beginner, and then 4 maybe intermediate or advanced. I'm not sure how to mark that one without doing it myself. But it doesnt look as hard as like, bombing up a slope which can be either advanced or expert depending on the surroundings.